README.md

```diff
@@ -24,13 +24,13 @@ You can compile the binary by doing `script/setup` and `script/compile`. Golang
   OR
   docker-gc (-command=ttl) (-interval=INTERVAL_IN_SECONDS) (-images_ttl=DURATION) (-containers_ttl=DURATION) for continuous cleanup based on image/container TTL
   OR
-  docker-gc (-command=diskspace) (-interval=INTERVAL_IN_SECONDS) (-high_disk_space_threshold=PERCENTAGE) (-low_disk_space_threshold=PERCENTAGE) for continuous cleanup based on used disk space
+  docker-gc (-command=diskspace) (-interval=INTERVAL_IN_SECONDS) (-high_disk_space_threshold=PERCENTAGE) (-low_disk_space_threshold=PERCENTAGE) (-containers_ttl=DURATION) for continuous cleanup based on used disk space
```

Comment:
I'm slightly confused by the grammer being used. Are you following a convention? I'm more used to this format:
```
docker-gc -command=diskspace -interval=<INTERVAL_IN_SECONDS> -high_disk_space_threshold=<PERCENTAGE> -low_disk_space_threshold=<PERCENTAGE> [-containers_ttl=<DURATION>]
```
where the contents of `<>` are to be filled in by the user, and `[]`'d parameters are optional.

Does the convention you are following have a method of specifying optional parameters?